src/common.ts

```diff
@@ -44,6 +44,12 @@ export type SortKey = string;
 export type WalletAddress = string;
 export type TransactionId = string;
 
+export type DataProtocolTransaction = {
+  id: string;
+  tags: { name: string; value: string }[];
+  data: number;
+};
```

Comment:
does arbundles or arweave-js export a type like this? ideally the SDK doesn't have to create it's own types for common ones

Reply:
Yes - updated.

Uses Pick with GQLNodeInterface.